src/REPLMode.jl

```diff
@@ -28,28 +28,18 @@ const cmds = Dict(
     "?"         => CMD_HELP,
     "status"    => CMD_STATUS,
     "st"        => CMD_STATUS,
-    "."         => CMD_STATUS,
-    "search"    => CMD_SEARCH,
-    "find"      => CMD_SEARCH,
-    "/"         => CMD_SEARCH,
     "add"       => CMD_ADD,
-    "install"   => CMD_ADD,
-    "+"         => CMD_ADD,
     "rm"        => CMD_RM,
     "remove"    => CMD_RM,
-    "uninstall" => CMD_RM,
-    "-"         => CMD_RM,
     "up"        => CMD_UP,
     "update"    => CMD_UP,
-    "upgrade"   => CMD_UP,
```

Comment:
Not sure if we should get rid of this one: the operation it's doing is an upgrade after all. I had proposed using the distinction between `update` and `upgrade` to differentiate getting a minor update versus doing a potentially-breaking `upgrade` but people didn't like the subtlety of the difference, which is why I made them aliases of each other, abbreviated to `up` for either one.

Reply:
Yep, the only potentially annyoing thing about keeping both is that right now `up\tab` shows both. We could canonicalize that to `update` though.

Reply:
Sure it does an `upgrade` and `remove` does an `uninstall` but keeping a bunch of different verbs that are almost synonyms seems unnecessary. One short form and one long form should be enough.

Reply:
Fair enough.

Reply:
Agreed; whenever I see both `update` and `upgrade` I get flashbacks to `brew` where they actually mean different things.